app/views/books/show.html.erb

```diff
@@ -1,4 +1,27 @@
-<h1>Top Three Reviews</h1>
+<h1><%= @book.title %></h1>
+<section>
+  <section class="book_image">
+    <img src="<%= @book.image %>" alt="image">
+  </section>
+  <ul>
+    <li>Author(s): <%= @book.lists_authors(@book) %></li>
+    <li>Pages: <%= @book.pages %></li>
+    <li>Year Published: <%= @book.year_pub %></li>
+  </ul>
+</section>
+<section class="show_reviews">
+  <h2>Reviews</h2>
+  <% @book.reviews.each do |review| %>
+  <section id="review">
```

Comment:
I'm wondering if we should make these id's more specific, such as book_show_reviews (book for book resources, show for show page and review for the section)?  I'm just thinking ahead to when we style and the possibility of having multiple CSS selectors across the whole site with the same name...